assets/src/js/app.js

```diff
@@ -0,0 +1,45 @@
+document.addEventListener('DOMContentLoaded', function(){
```

Comment:
IMO, ideally, it should be a class to increase readability and scalability.
But will do as a test widget 🙂